webapp/actions/channel_actions.jsx

```diff
@@ -191,3 +191,19 @@ export function loadChannelsForCurrentUser() {
     AsyncClient.getChannels();
     AsyncClient.getMyChannelMembers();
 }
+
+export function updateChannel(channel) {
+    Client.updateChannel(
+        channel,
+        () => {
+            AsyncClient.getChannel(channel.id);
+            this.handleHide();
```

Comment:
This function is from the react component and does not exists in the actions file